tests/native/test_armv7unicorn.py

```diff
@@ -27,8 +27,141 @@
 to make sure symbolic values get properly concretized.
 """
 
+assembly_cache = {
```

Comment:
```suggestion
# This is a cache of assembled instructions.
# This exists so that Manticore's tests can run without requiring that the
# Keystone dependency be installed.
# If additional test cases are added that require new instructions, this cache
# will need to be updated.
assembly_cache = {
```